smart-contracts/contracts/airdrop/src/admin.rs

```diff
@@ -0,0 +1,243 @@
+use std::string::String;
+
+use cosmwasm_std::{DepsMut, Env, Response, StdError, Uint128};
+use cw20_base::contract::query_balance;
+use cw_asset::Asset;
+
+use crate::helpers::get_total_in_user_info;
+use crate::state::{AirdropConfig, AIRDROP_CONFIG, USER_INFO};
+use crate::AirdropErrors;
+
+pub fn execute_update_airdrop_config(
+    deps: DepsMut,
+    env: Env,
+    config: AirdropConfig,
+) -> Result<Response, AirdropErrors> {
+    // Check various conditions to validate the airdrop configuration update
+
+    // Check if the start height and end height are not zero,
+    // indicating a valid airdrop window
+    if config.start_height != 0 && config.end_height != 0 {
+        // Check if the current block height is less than the start height
+        // and if the start height is less than the end height
+        if env.block.height < config.start_height && config.start_height < config.end_height {
```

Comment:
These block height checks are hard to read, a comment on what you're trying to achieve with them would be, such as 
```
start_height and end_height should always be in the future
```

Reply:
Sure. Adding more comments soon.